apps/arweave/src/ar.erl

```diff
@@ -541,6 +547,14 @@ parse_cli_args(["cm_exit_peer", Peer | Rest], C) ->
 			io:format("Peer ~p is invalid.~n", [Peer]),
 			parse_cli_args(Rest, C)
 	end;
+parse_cli_args(["pool_server" | Rest], C) ->
+	parse_cli_args(Rest, C#config{ pool_server = true });
+parse_cli_args(["pool_client" | Rest], C) ->
```

Comment:
In a CM cluster would both the CM Exit Node and CM Miner have pool_client = true?

Reply:
Yep, is_pool_client implies we need to fetch jobs from the pool